deploy/backplane/lpsre/hypershift/management-cluster/10-lpsre-management-cluster-project.ClusterRole.yml

```diff
@@ -1,7 +1,7 @@
 apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRole
 metadata:
-  name: backplane-lpsre-management-cluster-project
+  name: backplane-lpsre-management-cluster-cluster
```

Comment:
Very minor nit (won't affect any functionality), but the filename could maybe be renamed to `10-lpsre-management-cluster-cluster-cluster.ClusterRole.yml`

Reply:
fixed :)